js/main.js

```diff
@@ -0,0 +1,119 @@
+const form = document.querySelector('.form-option');
+const searchBar = document.querySelector('.search-bar');
+const searchButton = document.querySelector('.search-button');
+let resultsToDisplay;
+
+function fetchAPI() {
```

Comment:
FYI: I would rename to something like `fetchVolumes` to make it more clear! Eventually, you could even have 50+ fetch requests going on!